manifest.go

```diff
@@ -154,10 +156,10 @@ func (m Manifest) GeneratePreloadModules(name string) string {
 		}
 
 		if chunk.File != "" {
-			sb.WriteString(`<script type="preloadmodule" src="`)
+			sb.WriteString(`<link rel="modulepreload" href="`)
 			sb.WriteString("/")
 			sb.WriteString(chunk.File)
-			sb.WriteString(`"></script>`)
+			sb.WriteString(`">`)
```

Comment:
Good catch. How did that even work?

Reply:
I think before it just wasn't working - but because the preload only improves performance we wouldn't have noticed especially running locally